mlocal/frags/Makefile.stub

```diff
@@ -34,16 +34,15 @@ dist:
 .PHONY: unit-test
 unit-test:
 	@echo " TEST sudo go test [unit]"
-	$(V)cd $(SOURCEDIR) && sudo -E $(GO) test $(GO_MODFLAGS) -count=1 -timeout=20m -tags "$(GO_TAGS)" -failfast -cover -race \
+	$(V)$(SOURCEDIR) && sudo -E scripts/go-test \
```

Comment:
missing `cd`

Reply:
LOL -- I had removed the whole cd, thought better about it and added it back, but botched it, obviously...

thanks!